subsystem/subsystem-core/pom.xml

```diff
@@ -111,6 +111,11 @@
             <version>1.2.101.v20150831-1342</version>
         </dependency>
         <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
Or alternatively we could consider moving Subsystems up to Java 7, which contains Files.createTempDirectory() APIs. I think currently Subsystems is on Java 6, but I don't see many reasons not to move to Java 7. 

Personally I would not be so happy adding a dependency to Guava for subsystems core functionality, since this might interfere with Guava dependencies that subsystems users themselves might have.